druid/examples/game_of_life.rs

```diff
@@ -12,10 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#![allow(clippy::unreadable_literal)]
+
```

Comment:
Wow, that's odd. Does not happen without `--target wasm32-unknown-unknown` :unamused: 
I could not find out why clippy does that tho

Reply:
I could be wrong but my impression was that the examples were actually not checked by clippy before, but since the `druid-wasm-examples` is now in the workspace, they all get checked with `cargo clippy --all -- -D warnings` with or without `--target wasm32-unknown-unknown`. I remember having to make some other tweaks to the examples to satisfy clippy.